.github/workflows/ci.yml

```diff
@@ -21,15 +21,41 @@ jobs:
           toolchain: stable
       - run: RUSTFLAGS="-D warnings" cargo build
 
+  examples:
+    name: Cargo Examples
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        features:
+          - libftd2xx
+          - ftdi
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - run: |
+          sudo apt-get update
+          sudo apt-get install -y libftdi1 libftdi1-dev pkg-config
```

Comment:
These two steps should occur in a separate line only for `ftdi`, that way it skips this step for `libftd2xx`.

```
- if ${{ matrix.features == 'ftdi' }}
  run: |
    sudo apt-get update
    sudo apt-get install -y libftdi1 libftdi1-dev pkg-config
```

Same for the added features in `test`.

Reply:
done